validator/main.py

```diff
@@ -32,14 +32,9 @@ class BiasCheck(Validator):
     def __init__(
         self,
         threshold: float = 0.9,
-        on_fail: Optional[Union[str, Callable]] = None,
+        **kwargs,
```

Comment:
I think this would be a breaking change since `on_fail` is going fro a positional argument to a keyword only argument.